src/server/pkg/collection/collection.go

```diff
@@ -80,7 +80,7 @@ func (c *collection) indexDir(index Index, indexVal string) string {
 	indexDir := c.prefix
 	// remove trailing slash
 	indexDir = strings.TrimRight(indexDir, "/")
-	return fmt.Sprintf("%s__index_%s/%s", indexDir, index, indexVal)
```

Comment:
I think this will require a migration, right?  I think we actually found this issue before (i.e. that `indexVal` is not printed correctly) or a similar one, but decided to not fix it because it'd break existing deployments.